src/harness/reference_models/interference/aggregate_interference.py

```diff
@@ -253,13 +246,11 @@ def calculateAggregateInterferenceForPpa(ppa_record, pal_list, cbsd_list):
   Args:
     ppa_record: PPA protection entity
     pal_list: list of PAL records
-    cbsd_list: list of CBSD objects containing registrations and grants
+    cbsd_list: list of CbsdGrantInfo objects containing registrations and grants
```

Comment:
Elsewhere in come cleanup code, all those are called simply 'grants' or 'cbsd_grants'.
But do not worry on this in this PR, I will make a cleanup PR to finalize those cosmetic things a bit later to try having most of the code consistent.